src/test/java/uk/nhs/tis/trainee/notifications/migration/ResetLtftSubmittedTraineeHistoryTest.java

```diff
@@ -74,17 +72,13 @@ void setUp() {
 
   @Test
   void shouldResetHistoryWithLtftTraineeSubmittedType() {
-    History history = History.builder()
-        .type(LTFT_SUBMITTED_TRAINEE)
-        .build();
-    history = mongoTemplate.save(history);
+    mongoTemplate.save("{ \"type\": \"LTFT_SUBMITTED_TRAINEE\" }", "History");
 
     migrator.migrate();
 
-    History migratedHistory = mongoTemplate.findById(history.id(), History.class);
-
-    assertThat("Unexpected missing history.", migratedHistory, notNullValue());
-    assertThat("Unexpected type.", migratedHistory.type(), is(LTFT_SUBMITTED));
+    Query query = Query.query(Criteria.where("type").is("LTFT_SUBMITTED_TRAINEE"));
+    assertThat("Unexpected unmigrated history.",
+        mongoTemplate.findAllAndRemove(query, "History").isEmpty(), is(true));
```

Comment:
This is a slightly odd looking way of verifying the record has been changed, but allows one to query without stipulating a class